flowmachine/tests/test_query_object_construction.py

```diff
@@ -355,3 +403,78 @@ def test_invalid_sampling_params_raises_error(sampling, message):
     with pytest.raises(ValidationError, match=message) as exc:
         _ = FlowmachineQuerySchema().load(query_spec)
     print(exc)
+
+
+def test_unmatching_spatial_unit_raises_error_daily(location_list_unmatched_agg):
```

Comment:
These tests still reference the now-removed fixture (although they don't use it)